editor/components/post-permalink/index.js

```diff
@@ -53,7 +53,7 @@ class PostPermalink extends Component {
 				<Dashicon icon="admin-links" />
 				<span className="editor-post-permalink__label">{ __( 'Permalink:' ) }</span>
 				<Button className="editor-post-permalink__link" href={ link } target="_blank">
-					{ link }
+					{ decodeURI( link ) }
```

Comment:
Can you explain the choice of `decodeURI` over `decodeURIComponent` ?

Reply:
I may be wrong but I use `decodeURI` when I have to decode the whole URL and `decodeURIComponent` when decoding a slice of the URL (just a value of a query string for instance)